docs/builds.rst

```diff
@@ -3,6 +3,11 @@
 Builds
 ======
 
+You might want to take a look at the following pages before continue reading:
```

Comment:
"... continue reading" -> exploring about automated builds.

Reply:
Changed it for the other pages too (experiments, revisions, ...)